tensorflow/python/ops/ragged/ragged_math_ops.py

```diff
@@ -707,7 +707,8 @@ def reduce_variance(input_tensor: ragged_tensor.Ragged,
         input_tensor, name='input_tensor')
     if input_tensor.dtype.is_complex:
       raise ValueError(
-          'reduce_variance is not supported for RaggedTensors with complex dtypes.'
+          'reduce_variance is not supported for RaggedTensors with complex'
```

Comment:
nit: please add a space either after the word "complex" or  in the next line before "dtypes". Otherwise it means the constructed string would have the word complexdtypes (without space)

Reply:
Done!